.gitignore

```diff
@@ -22,3 +22,4 @@ static/zz_generated_assets.go
 docs/cli
 site
 k0sctl.yaml
+*.swp
```

Comment:
I'd suggest to put those things into `$XDG_CONFIG_HOME/git/ignore`, so we don't have to accumulate all sorts of OS and/or editor specific stuff here.

Reply:
Removed changes to this file

Reply:
Added signoff to latest commit.